docs/user/openstack/install_upi.md

```diff
@@ -156,25 +148,42 @@ python3 -m pip install --upgrade pip
 python3 -m pip install yq openstackclient openstacksdk netaddr
 ```
 
+Make sure, that none of copied playbooks contains an alias for ansible
+utility filters:
+
+```sh
+for playbook in *.yaml; do
+    sed -i -e 's/ansible.utils.//g' $playbook
+done
+```
+
 ### Fedora
 
 This command installs all required dependencies on Fedora:
 
 ```sh
-sudo dnf install python3-openstackclient ansible-core python3-openstacksdk python3-netaddr
+sudo dnf install python3-openstackclient python3-netaddr ansible-collections-openstack ansible-collection-ansible-utils
 ```
 
 [ansible-upi]: ../../../upi/openstack "Ansible Playbooks for Openstack UPI"
 
 ## Ansible Collections
 
-The Ansible Collections are not packaged (yet) on recent versions of OSP and RHEL when `ansible-core` is
-installed instead of Ansible 2.9. So the collections need to be installed from `ansible-galaxy`.
+Alternatively you could use ansible-galaxy to install required collections,
```

Comment:
I'm a bit lost here. Alternatively to what? The title starts with two `##`; the previous title with the same indentation is `## Install Ansible`.

Is this paragraph an alternative to what's recommended for `### Fedora`?

Reply:
I'll remove whole ansible-galaxy section as it is confusing, and might do more harm than good in current state.